### PR TITLE
Add support for gha image scans

### PR DESCRIPTION
**Pull request by tsmithv11**
**By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

## Description

Adding support for running container image scans in GitHub Actions

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my feature, policy, or fix is effective and works
- [x] New and existing tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by tsmithv11** [comment]
I need to add flags that are relevant

File: github_action_resources/entrypoint.sh [hunk 1]
```diff
@@ -101,7 +103,10 @@ echo "BC_REPOSITORY_URL="${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}""
 
 echo "running checkov on directory: $1"
 
-if [ -n "$API_KEY_VARIABLE" ]; then
+if [ -n "$API_KEY_VARIABLE" ] && [ -n "$DOCKER_IMAGE"]; then
+  echo "checkov --bc-api-key XXXXXXXXX-XXX-XXXXX --branch $GIT_BRANCH --repo-id $GITHUB_REPOSITORY $DOCKER_IMAGE_FLAG $DOCKERFILE_PATH_FLAG" #Unsure what other flags should work
```
Comment (schosterbarak) [inline github_action_resources/entrypoint.sh:107]: hard coded xxx api key?

Comment (tsmithv11) [reply github_action_resources/entrypoint.sh:107]: That's just for the echo. I copied what the previous one did (see below). I'll update both.

**Review by gruebel — APPROVED** [review]
🍻

**Review by Eliran-Turgeman — APPROVED** [review]

**Pull request merged**